src/stories/Seekers/CanAddSeeker/endpoint.spec.js

```diff
@@ -54,8 +54,11 @@ describe("Test API Seekers/CanAddSeeker", () => {
           company_type: "start-up",
           company_employee_strenget: "50-100",
           current_salary: {
-            currency: {},
-            range: {},
+            currency: "INR",
+            range: {
+              from: "500000",
+              to: "700000",
+            },
           },
           note_for_recruiter: "",
           notice_period: "10-20 days",
```

Comment:
we won't be able to query it `notice_period: "10-20 days` it can't be a range. It should be in days.